user_groups/user_groups.md

```diff
@@ -311,7 +365,7 @@ POST /v1/usergroups/{userGroupId}/permissions/{resourceId}
 | Name | Type | Description |
 | ------------- |------------- | -------------|
 | userGroupId | `string` | The user group id to request. |
-| resourceId | `string` | The resource id to add or update. |
+| resourceId | `string` | The resource id to update. |
```

Comment:
A resource id to set up the new permission to.

Reply:
updated